imports/plugins/core/ui/client/containers/tagListContainer.js

```diff
@@ -241,7 +241,7 @@ function composer(props, onData) {
 
   if (props.product) {
     if (_.isArray(props.product.hashtags)) {
-      tags = _.map(props.product.hashtags, (id) => Tags.findOne(id));
+      tags = Tags.find({ _id: { $in: props.product.hashtags } }).fetch();
```

Comment:
I found myself doing a `compact` on tags but then realized there are N queries, which could return `null`. Changing to a single query eliminates the `null`s for us.
(this pattern was used in a few places. I replaced them all)